Xamarin.Essentials/SaveToGallery/SaveToGallery.ios.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Threading.Tasks;
+using Foundation;
+using UIKit;
+
+namespace Xamarin.Essentials.SaveToGallery
+{
+    public static partial class SaveToGallery
+    {
+        static Task PlatformSaveImageAsync(byte[] data, string filename, string albumName)
+        {
+            var image = new UIImage(NSData.FromArray(data));
+            image.SaveToPhotosAlbum((UIImage img, NSError error) => { });
```

Comment:
`UIImage.SaveToPhotosAlbum` does not save metadata

Reply:
yes I know, I'm trying to find the right cross-platform solution

Reply:
Might be a better option (that's for iOS). 
```
var library = new ALAssetsLibrary();
var albumSave = await library.WriteImageToSavedPhotosAlbumAsync(cgImage, meta);
var albumSave = await library.WriteVideoToSavedPhotosAlbumAsync(new NSUrl(path));
```